src/cml.js

```diff
@@ -469,15 +469,19 @@ class CML {
       userName = GIT_USER_NAME,
       remote = GIT_REMOTE
     } = opts;
+    let { fetchDepth = 1 } = opts;
```

Comment:
wait won't this now by default truncate non-shallow clones (as per [SO#46004595](https://stackoverflow.com/a/46004595))?

i.e. `cml ci` will now undo `actions/checkout.with.fetch-depth: 0`?

Reply:
it does appear so. I'll do a follow up

Reply:
https://github.com/iterative/cml/pull/1246